### PR TITLE
:sparkles: Add initialization of OneSignal

### PR DESCRIPTION
**Pull request by issa-diallo**
The SDK is correctly integrated, and OneSignal initialization is error-free

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -1,21 +1,19 @@
...
-    <meta name="theme-color" content="#000000" />
-    <meta
-      name="description"
-      content="Patient management app for nurses"
-    />
-    <link rel="apple-touch-icon" href="%PUBLIC_URL%/apple-touch-icon.png" />
-    <!--
+
+<head>
+  <meta charset="utf-8" />
```
Comment (AndreMiras) [inline public/index.html:5]: Why is the indentation messed up?
Maybe do a `npx prettier --write public/` until we set the formatter correctly

File: public/index.html [hunk 2]
```diff
@@ -24,12 +22,22 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
-    <title>MyNotif</title>
-  </head>
-  <body>
-    <noscript>You need to enable JavaScript to run this app.</noscript>
-    <div id="root"></div>
-    <!--
+  <script src="https://cdn.onesignal.com/sdks/OneSignalSDK.js" defer></script>
```
Comment (AndreMiras) [inline public/index.html:25]: Wait don't we have a better way to integrate that with React?

File: src/services/oneSignal/OneSignalManager.tsx [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+        await OneSignal.init({
+          appId: process.env.REACT_APP_ONE_SIGNAL_ID as string,
+          allowLocalhostAsSecureOrigin: true
+        })
+      } catch (error) {
+        console.error('Error initializing OneSignal:', error)
+      }
+    }
+    // eslint-disable-next-line
+    initializeOneSignal()
```
Comment (AndreMiras) [inline src/services/oneSignal/OneSignalManager.tsx:17]: As usual, be more specific

Comment (issa-diallo) [reply src/services/oneSignal/OneSignalManager.tsx]: fixed

File: public/index.html (hunk 2 above)
Comment (issa-diallo) [reply public/index.html]: I've integrate via yarn add

**Review by AndreMiras — APPROVED** [review]

**Pull request merged**